initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -13,6 +13,19 @@
             <xs:attribute name="warnIfFreeMemoryUnder" type="xs:nonNegativeInteger" use="optional" default="5" hpcc:displayName="Warn Free Memory" hpcc:group="Monitoring" hpcc:tooltip="Available memory under this value is flagged as warning in monitoring output"/>
         </xs:attributeGroup>
 
+        <xs:attributeGroup name="elkintegration">
+            <xs:attribute name="integrateKibana" type="xs:boolean" use="required"  default="true" hpcc:tooltip="Declare if Kibana dashboard to be integrated in ECLWatch"/>
```

Comment:
@kenrowland  the default value should actually be false.

Reply:
@rpastrana Changed to false in latest commit. Please review.